src/components/signinForm/SigninForm.js

```diff
@@ -0,0 +1,53 @@
+/* tslint:disable:no-empty */
+import FormControl from '../../shared/FormControl';
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import './SigninForm.scss';
```

Comment:
Line 5 typo please, capitalize the i in `SigninForm`
eg`import './SignInForm.scss';`